src/lib/components/event/event-details-row-expanded.svelte

```diff
@@ -25,15 +25,27 @@
   export let inline = false;
 
   const { workflow, namespace } = $page.params;
+  $: codeBlockValue = getCodeBlockValue(value);
+  $: stackTrace =
+    codeBlockValue?.stackTrace || codeBlockValue?.cause?.stackTrace;
```

Comment:
We could do that instead! I was hoping we could avoid that as it looks like there aren't that many places it could be, but I definitely could have missed some possibilities 😅